gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/GaaSObservabilityEventProducer.java

```diff
@@ -137,26 +138,29 @@ public Attributes getEventAttributes(GaaSObservabilityEventExperimental event) {
    * Emits the GaaSObservabilityEvent with the mechanism that the child class is built upon e.g. Kafka
    * @param event
    */
-  abstract protected void sendUnderlyingEvent(GaaSObservabilityEventExperimental event);
+  abstract protected void sendUnderlyingEvent(GaaSObservabilityEvent event);
 
   /**
    * Creates a GaaSObservabilityEvent which is derived from a final GaaS job pipeline state, which is combination of GTE job states in an ordered fashion
    * @param jobState
    * @return GaaSObservabilityEvent
    */
-  private GaaSObservabilityEventExperimental createGaaSObservabilityEvent(final State jobState) {
+  private GaaSObservabilityEvent createGaaSObservabilityEvent(final State jobState) {
     Long jobStartTime = jobState.contains(TimingEvent.JOB_START_TIME) ? jobState.getPropAsLong(TimingEvent.JOB_START_TIME) : null;
     Long jobEndTime = jobState.contains(TimingEvent.JOB_END_TIME) ? jobState.getPropAsLong(TimingEvent.JOB_END_TIME) : null;
     Long jobOrchestratedTime = jobState.contains(TimingEvent.JOB_ORCHESTRATED_TIME) ? jobState.getPropAsLong(TimingEvent.JOB_ORCHESTRATED_TIME) : null;
     Long jobPlanningPhaseStartTime = jobState.contains(TimingEvent.WORKUNIT_PLAN_START_TIME) ? jobState.getPropAsLong(TimingEvent.WORKUNIT_PLAN_START_TIME) : null;
     Long jobPlanningPhaseEndTime = jobState.contains(TimingEvent.WORKUNIT_PLAN_END_TIME) ? jobState.getPropAsLong(TimingEvent.WORKUNIT_PLAN_END_TIME) : null;
+    String edgeName = jobState.getProp(TimingEvent.FlowEventConstants.FLOW_EDGE_FIELD, "");
+    String[] edgeNameParts = edgeName.split(BaseFlowGraphHelper.FLOW_EDGE_LABEL_JOINER_CHAR);
```

Comment:
unfortunately this format
```
<src_node>_<dest_node>_<edge_id>
```
is not always correctly parsable, due to some nodes containing `_`.

is there any way to obtain the constituent parts which haven't already been joined (into this not-necessarily-reversable) form?